app/experimenter/nimbus-ui/src/components/PageResults/ConfidenceInterval/index.tsx

```diff
@@ -6,108 +6,89 @@
 // TODO: EXP-638 remove this ^
 
 import React from "react";
-import { SIGNIFICANCE } from "../../../lib/visualization/constants";
 
-const renderBounds = (lower: number, upper: number, significance: string) => {
-  const padding = <div className="col p-0" />;
-  const numbers = (
-    <>
-      <div
-        className={`${significance}-significance col text-left p-0 h6 font-weight-normal`}
-      >
-        {lower}%
-      </div>
-      <div
-        className={`${significance}-significance col text-right p-0 h6 font-weight-normal`}
-      >
-        {upper}%
-      </div>
-    </>
-  );
-
-  switch (significance) {
-    case SIGNIFICANCE.NEGATIVE:
-      return (
-        <>
-          {numbers}
-          {padding}
-          {padding}
-        </>
-      );
-    case SIGNIFICANCE.POSITIVE:
-      return (
-        <>
-          {padding}
-          {padding}
-          {numbers}
-        </>
-      );
-    case SIGNIFICANCE.NEUTRAL:
-      return (
-        <>
-          {padding}
-          {numbers}
-          {padding}
-        </>
-      );
-  }
-};
-
-const renderLine = (significance: string) => {
-  let firstLine, secondLine, thirdLine;
-  firstLine = secondLine = thirdLine = "col";
-
-  if (SIGNIFICANCE.NEGATIVE === significance) {
-    firstLine = "col-md-1";
-    secondLine = "col-md-4";
-  }
-
-  if (SIGNIFICANCE.POSITIVE === significance) {
-    secondLine = "col-md-4";
-    thirdLine = "col-md-1";
-  }
-
-  return (
-    <>
-      <div className={`${firstLine} border-bottom border-dark border-3`} />
-      <div
-        className={`${secondLine} md-4 ml-md-auto py-2 ${significance} mb-n2`}
-      />
-      <div className={`${thirdLine} md-1 border-bottom border-dark border-3`} />
-    </>
-  );
-};
-
-const renderTick = (significance: string) => {
-  let position = "py-2";
-  if (SIGNIFICANCE.NEUTRAL === significance) {
-    position = "py-1 mt-2";
-  }
+const renderBounds = (
+  lower: number,
+  upper: number,
+  leftPercent: number,
+  barWidth: number,
+  significance: string,
+  buffer: number,
```

Comment:
I think it'd make sense to take `const BUFFER = 5` out of `ConfidenceInterval` and stick it above `renderBounds`, that way you don't have to pass it as a prop here.